cmd/notary/tuf.go

```diff
@@ -83,6 +83,12 @@ var cmdTUFVerifyTemplate = usageTemplate{
 	Long:  "Verifies if the data passed in STDIN is included in the remote trusted collection identified by the Globally Unique Name.",
 }
 
+var cmdTUFDeleteTemplate = usageTemplate{
+	Use:   "delete [ GUN ]",
+	Short: "Deletes all content for a trusted collection",
+	Long:  "Deletes all content for a trusted collection identified by the Globally Unique Name, can delete both local and remote data based on flags.",
```

Comment:
(I don't feel strongly at all on the wording, since the flag specifies additional detail - mainly was thinking there should be a semicolon or period between the two parts of the sentence :))